pkg/services/star/starimpl/star.go

```diff
@@ -0,0 +1,39 @@
+package starimpl
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/star"
+)
+
+type serviceImpl struct {
+	starStore store
+}
+
+func ProvideService(sqlstore sqlstore.Store) star.Service {
+	m := &serviceImpl{starStore: newStarStore(sqlstore)}
```

Comment:
I wonder, how "star service" becomes an "m" if that's one of a few letters not present in any of those words? 🤔